### DISCUSSION TITLE
Week 13- Project Completion and Reflection

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#238] [General]
**Completed Project Page and Portfolio:**
Github Portfolio Code: https://github.com/CJ3315/Portofolio-Project
Live Portfolio Published: https://cj3315.github.io/Portofolio-Project/

**Reflection**
Okay with this project came a lot of handwork and headaches but I am happy to say that in the end, it was a very fulfilling project and I am happy with the end result. 
I was able to create a functioning portfolio website with the use of **HTML** by adding different sections, headings, images and video links to create an easy-to-navigate showcase of some of my previous work. 
Through the use of **CSS**, I created different background colors and fonts along with different control buttons to click on for navigation to different sections. 
I was able to add functionality and identifiers through my minimal use of **JavaScript.**
The element that I am most proud of is my use of **SCSS** to create different transition, hover, and interactive functions to make my site look professional and pleasing. 
I feel that the **usability** of my site is adequate and easy to discover and follow. 

The one area that I would like to work on in the future would be **accessibility.** My page does not translate overall well to mobile devices and is intended to be viewed on a lap/desktop. I would like to fix this in the future, along with the addition of a light/dark mode and potentially a zoom function. 

Overall I am very pleased with this work and hope to add to it in the future.

**Comment by gabebott** [comment]
Hey CJ, I just took a look at your site and that looks awesome! I think you did a really good job of creating a visually enticing site while also making it very accessible. Great job!

**Comment by kassyoveson** [comment]
You did a great job CJ. I really like how your site turned out. You did great with the color contrasts. For another one of my classes I used Adobe DreamWeaver. That program has built in accessibility for devices. It was nice to have this class, so I actually could read code that was in the template I was using. More proof that I actually learned something in this class. (Just kidding) Have a great rest of your summer.
